electron/scripts/carer.js

```diff
@@ -53,16 +53,41 @@ function populateCarerTableElements(jdata) {
     return [hist_tes, curr_tes];
 }
 
-// Just an example to show off that a function can be called when you check the box.
-// This should be used to end the child carer relationship, i.e. set the end date
-function exampleCheckboxFunction(element, idx, entry) {
-    let checkedText = element.checked ? "CHECKED" : "UNCHECKED";
-    alert("You " + checkedText + " the checkbox for row " + idx + " which is the row with child " + entry.caregiver_english_name + " who has ID " + entry.caregiver_id);
+function endChildCaregiverRelationship(element, idx, row_obj) {
+    if (!element.checked) {
+        alert("This checkbox only supports ENDING a child/caregiver relationship (cannot uncheck)")
+        element.checked = true;
+    }
+    let today = new Date();
+    let dd = today.getDate();
+    let mm = today.getMonth() + 1; //January is 0
+    let yyyy = today.getFullYear();
+    if(dd < 10) { dd = '0' + dd; }
+    if(mm < 10) { mm = '0' + mm; }
+    today = yyyy + '-' + mm + '-' + dd;
```

Comment:
FYI there are many library functions that do this, here is an example from jQuery:
let today = jQuery.datepicker.formatDate("yy-mm-dd",new Date());
What you have is fine though.